### PR TITLE
[dev] -  AB#65920 - Implement Header Search Bar

### PR DESCRIPTION
**Pull request by bittu1028**

### CODE REVIEW

**Comment by bittu1028** [comment]
still need to figure out and discard these search term value

File: src/modules/animatedHeader/animatedHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    {
+      shadowOpacity: 0.3,
+      elevation: 8,
+      top: DeviceUtils.isAndroid() ? 56 : 0,
+      transform: [{translateY}],
+    },
+    containerStyle,
+  ]);
+
+  const wrrapperStyle = StyleSheet.flatten([{opacity}, itemStyle]);
```
Comment (ajaybushel) [inline src/modules/animatedHeader/animatedHeader.tsx:86]: misspelled wrapper

File: src/modules/animatedHeader/animatedHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  });
+  return {opacity, translateY};
+};
+
+const StyledContainer = Styled.View`
+    position: 'absolute';
+    left: 0;
+    right: 0;
+    shadowColor: ${({theme}) => theme.colors.grayOne};
+    box-shadow: 4.65px 0px 4px black;
```
[the remark this answers is not in the document]
Comment (bittu1028) [reply src/modules/animatedHeader/animatedHeader.tsx:40]: @ajaybushel I will confirm this i was little bit confused with styled-components

File: src/modules/animatedHeader/animatedHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      top: DeviceUtils.isAndroid() ? 56 : 0,
+      transform: [{translateY}],
+    },
+    containerStyle,
+  ]);
+
+  const wrrapperStyle = StyleSheet.flatten([{opacity}, itemStyle]);
+
+  oldOpacity = opacity;
+  oldTranslateY = translateY;
```
Comment (ajaybushel) [inline src/modules/animatedHeader/animatedHeader.tsx:89]: can we call this oldOpacity and newOpacity OR prevOpacity and newOpacity?

File: src/modules/animatedHeader/animatedHeader.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (bittu1028) [reply src/modules/animatedHeader/animatedHeader.tsx]: done

File: src/modules/animatedHeader/animatedHeader.tsx (hunk 1 above)
Comment (bittu1028) [reply src/modules/animatedHeader/animatedHeader.tsx]: fixed

File: src/modules/animatedHeader/animatedHeader.tsx (hunk 3 above)
Comment (bittu1028) [reply src/modules/animatedHeader/animatedHeader.tsx]: fixed

**Review by ajaybushel — APPROVED** [review]

**Review by raoufr — APPROVED** [review]

**Pull request merged**